### PR TITLE
Add "Copy Relative File Path" Option to Changed Files' Context Menu

### LINKED ISSUE #12589: Add "Copy Relative File Path" menu item
This menu item eases commit message writing and command line operations that are relative to the root of repository. The file explorer of Visual Studio Code has already featured this menu item for years.

### PR DESCRIPTION
**Pull request by uttiya10**
Closes #12589

## Description
Add the ability to copy a changed files' relative path

### Screenshots
image

## Release notes
Notes:
- [ADDED] Copy the Relative File Path of a Changed File

### CODE REVIEW

**Review by tidy-dev — CHANGES_REQUESTED** [review]
@uttiya10 Thank you for wanting to add this context menu!

This is looking great and tested it out great.

One request tho is that we also have the "Copy File Path" on the files in the history view. We would like to be consistent and have this "Copy Relative File Path", there as well. Should be simply adding the same thing to `selected-commit.tsx` file.

**Review by tidy-dev — CHANGES_REQUESTED** [review]
Thanks for adding that additional context menu @uttiya10!  It works great!

I have one super small request, bit of a nit, and then I think it should be good to go! Seeing as the "View on GitHub" menu item is not conditional, I think it makes senes to leave that logic separate and put the `{ type: 'separator' }` added in line 384 of `selected-commit.tsx` beneath the new context menu at line 370.

**Comment by uttiya10** [comment]
Ahh, I misunderstood the code 🤦‍♂️. I thought the `View on Github` menu item wasn’t always shown, so a separator is optional. However, its actually always shown but *not always enabled*. Will push an update soon.

**Review by tidy-dev — APPROVED** [review]
✨  Thank you so much for those changes and wanting to contribute to GitHub Desktop!

File: app/src/ui/changes/changes-list.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import {
   CopyFilePathLabel,
   RevealInFileManagerLabel,
   OpenWithDefaultProgramLabel,
+  CopyRelativeFilePathLabel,
 } from '../lib/context-menu'
```
Comment (ZmoraKaBeKa) [inline app/src/ui/changes/changes-list.tsx:27]:
```suggestion
 } from '../lib/context-menu'haha
```

**Pull request merged**